src/runtime-rs/crates/hypervisor/src/device/device_manager.rs

```diff
@@ -325,6 +330,21 @@ impl DeviceManager {
                 // No need to do find device for hybrid vsock device.
                 Arc::new(Mutex::new(HybridVsockDevice::new(&device_id, hvconfig)))
             }
+            DeviceConfig::ShareFsCfg(config) => {
+                // Try to find the sharefs device. If found, just return.
+                if let Some(device_id_matched) = self.find_device(config.shared_path.clone()).await
```

Comment:
I'm wondering why `find_device()` takes an owned string instead of a string slice, forcing all this `clone()`ing... But that's a case for another PR, ignore me here. ;-)

Reply:
Yeah，I will consider to refactor it future!